src/views/studio/MyVideosView/AddVideoView.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react'
+import styled from '@emotion/styled'
+import bgIllustration, { ReactComponent as WellIllustration } from '@/assets/well-blue.svg'
+import { Button, Text } from '@/shared/components'
+import { sizes, colors } from '@/shared/theme'
+import { css } from '@emotion/react'
+
+// for when there is absolutely no videos available
+export const AddVideoView = () => {
+  return (
+    <ContainerView>
+      <InnerContainerView>
+        <MessageView>
+          <Text variant="h3">Add your first video</Text>
+          <Subtitle variant="body2">
+            No videos available. Start the publisher journey by adding
+            <br />
+            you very first video to Joystream.
+          </Subtitle>
+        </MessageView>
+        <div>
+          <Button icon="video-camera">Upload video</Button>
+        </div>
+      </InnerContainerView>
+      <WellIllustration />
+    </ContainerView>
+  )
+}
+
+const ContainerView = styled.div`
+  height: 50vh;
+  margin: ${sizes(20)} auto 0;
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  > svg {
+    max-width: 650px;
+  }
+`
+
+const InnerContainerView = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+`
+
+const MessageView = styled.div`
+  display: flex;
+  flex-direction: column;
+  text-align: left;
+  margin-top: 48px;
+  margin-bottom: ${sizes(8)};
+`
+
+// for tabs
+export const AddVideo = () => {
```

Comment:
This won't really work for empty tabs since adding a video doesn't really guarantee that `unlisted` or `drafts` will fill up. For empty tabs maybe we should just show something like `Nothing to show`?

Reply:
<img width="1437" alt="Screenshot 2021-03-18 at 18 54 44" src="https://user-images.githubusercontent.com/10615263/111673672-81e9a680-881b-11eb-87b4-a7ac2d9eb05d.png">

What about this? I think it's useful to have the Upload video button in there as it might be something the user is looking for anyways, and it also doesn't imply adding a video will fill in the tab, thoughts?